src/chat/Database.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "KeywordResponses.hpp"
+
+namespace chat {
+
+/**
+ * Databases may be configured to link to one other database.
+ * This indicates that a Database does not have a link.
+ */
+constexpr inline auto NoDatabaseLink = -1;
+
+/**
```

Comment:
I'm hoping to kill Record and replace it with a vector of the KeywordResponses

Reply:
Cool! Thanks. If you keep the poitner stuff (you probably won't) be aware of pointer stability.